### PR TITLE
Bl/#1510 cacc graphs 4ci

### PR DESCRIPTION
**Pull request by bhavyalatha26**
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/1535)

### CODE REVIEW

**Comment by wrashid** [comment]
I just ran sflight 1k on this branch and found that the correct data is being written out to 0.caccCapacityStats.csv. However the graph generation code did not work (empty graph). It should look as follows (only blue dots): 

image

**Comment by bhavyalatha26** [comment]
ok @wrashid

**Comment by wrashid** [comment]
Test!

File: src/main/java/beam/physsim/jdeqsim/cacc/roadCapacityAdjustmentFunctions/Hao2018CaccRoadCapacityAdjustmentFunction.java [hunk 1]
```diff
@@ -118,4 +143,74 @@ private void writeCapacityStats(int iterationNumber,String statsData) {
...
         return  (writeInterval > 0 && iterationNumber % writeInterval == 0);
     }
+
+    /**
+     * A scattered plot that analyses the percentage of increase of road capacity observed for a given fraction of CACC enabled travelling on
+     * CACC enabled roads
+     * @param iterationNumber current iteration number
+     * @param caccCapacityIncrease data map for the graph
+     */
+    private void generateCapacityIncreaseScatterPlotGraph(int iterationNumber, Map<Double, Double> caccCapacityIncrease) {
```
Comment (wrashid) [inline src/main/java/beam/physsim/jdeqsim/cacc/roadCapacityAdjustmentFunctions/Hao2018CaccRoadCapacityAdjustmentFunction.java]: please extract to separate class

Comment (bhavyalatha26) [reply src/main/java/beam/physsim/jdeqsim/cacc/roadCapacityAdjustmentFunctions/Hao2018CaccRoadCapacityAdjustmentFunction.java]: ok @wrashid

**Pull request merged**